default-config.xml

```diff
@@ -66,6 +66,8 @@
         <property name="userAttributes">
             <map>
                 <!--This is now required because the parallelJobsNumber setting is not entirely reliable-->
```

Comment:
the comment on line 68 goes with the content on line 71, which is a bit confusing. I can see how my instructions were not super clear there.
can you make sure the comment stays directly above the line it applies to.